app/models/tenant.rb

```diff
@@ -292,6 +292,8 @@ def allowed?
   end
 
   def create_miq_product_features_for_tenant_nodes
+    return [] if !miq_region || MiqRegion.my_region_number != miq_region.region
```

Comment:
I don't understand how this would solve the problem you're seeing. This method is only called by an `after_create` hook, right? That isn't triggered when rows are replicated from remote regions so I don't see how this method would ever be called on an instance from a remote region in the global.